mongo/options/serverapioptions.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package options
+
+import "fmt"
+
+// ServerAPIOptions represents options used to configure the API version sent to the server
+// when running commands.
+//
+// Sending a specified server API version causes the server to behave in a manner compatible with that
+// API version. It also causes the driver to behave in a manner compatible with the driver’s behavior as
+// of the release when the driver first started to support the specified server API version.
+//
+// The user must specify a ServerAPIVersion if including ServerAPIOptions in their client. That version
+// must also be currently supported by the driver.
+type ServerAPIOptions struct {
```

Comment:
As a safety check, we do want "API" capitalized, per Go naming conventions, right? The spec capitalizes as "Api"

Reply:
It's a good question; I was also wondering if I should call this `ServerAPI` per the spec instead of `ServerAPIOptions`. Will discuss offline and update this comment.